### PR TITLE
DB-3787: Refactor block editor support and remove html-react-parser

### PR DESCRIPTION
**Pull request by enZane**
## What changes were made?
- Changed the implementation of the post and page rendering
- Removed html-react-parser dependency

## Where were the changes made?
- gatsby-wordpress-starter

## How have the changes been tested?
- locally

## Additional information

Don't forget to [add a changeset](https://github.com/pantheon-systems/decoupled-kit-js#generating-a-changeset) if needed!

### CODE REVIEW

File: starters/gatsby-wordpress-starter/src/components/post.jsx [hunk 1]
```diff
@@ -15,6 +14,8 @@ const Post = ({
     altText: featuredImage?.node?.altText || title,
   }
 
+  console.log({ previous, next })
```
Comment (CobyPear) [inline starters/gatsby-wordpress-starter/src/components/post.jsx:17]: 👀 stray console log?

**Review by CobyPear — CHANGES_REQUESTED** [review]
Just one stray console log, LGTM tho, block editor styles seem to be working well!

**Comment by CobyPear** [comment]
Oh also in this PR, can you update the `package-lock.json` to reflect the removed dependency?

File: starters/gatsby-wordpress-starter/src/components/post.jsx (hunk 1 above)
Comment (enZane) [reply starters/gatsby-wordpress-starter/src/components/post.jsx:17]: Oops :p

**Review by CobyPear — APPROVED** [review]

**Pull request merged**